runtimevar/runtimevar_test.go

```diff
@@ -193,6 +192,45 @@ func TestVariable(t *testing.T) {
 	}
 }
 
+var errFake = errors.New("fake")
+
+// fakeError implements driver.Watcher.
+// WatchVariable always returns a state with errFake, and Close
+// always returns errFake.
+type fakeErrorer struct {
```

Comment:
fakeErrorer sounds weird, especially it doesn't have a function called `fakeError`. How about just `fakeWatcher` or `errorWatcher`?

Reply:
Yeah, you are right. Changed to `erroringWatcher`. I also updated the similar fakes in `blob_test`, to be consistent.